.github/dependabot.yml

```diff
@@ -8,6 +8,12 @@ updates:
     target-branch: "main"
     labels:
       - "dependencies"
+    groups:
+      dev-dependencies:
+        exclude-patterns:
+         - "puppeteer"
+         - "@duckduckgo/*"
+         - "privacy-test-pages"
```

Comment:
Do these apply? They're normal dependencies? Is it just belt and braces in case we move them?

Reply:
Oh I guess `dev-dependencies` is just a group name rather than having distinct meaning. Should we rename the group name?

Reply:
Yeah, the idea is that we'll get individual PRs for `@duckduckgo/*` and `privacy-test-pages`, as those will be worth reviewing individually. Everything else will be bundled into a single version bump PR each week for `dev-dependencies`.

Reply:
Yeah fair, I'd still rename the group name given the collision with npm naming but whatever works 👍🏻